values.yaml

```diff
@@ -139,6 +139,27 @@ podDisruptionBudget: {}
   # maxUnavailable: 1
   # minAvailable: 2
 
+autoscaling:
+  enabled: false
+  apiVersion: v1
+  minReplicas: 1
+  maxReplicas: 2
+  targetCPUUtilizationPercentage: 75
+  targetMemoryUtilizationPercentage: 75 # available only on version v2
+  behavior: {} # available only on version v2
```

Comment:
Thanks for this comment to clarify why a user might set a value and see it being ignored.

With the suggested removal of the apiVersion prop, though, the comment would lose the benefit of being in close promixity to the version it is referencing and become ambiguous.

Proposed change to more directly identify the constraints of when the value would take effect: "available only on Kubernetes ≥1.23"
(i.e: when the autoscaling/v2 HPA was introduced)

Reply:
Updated, and I also updated values definitions inside `ReadME.md`.